schemas/oic.sec.dpmtype.json

```diff
@@ -13,12 +13,12 @@
           "description": "The value can be either 8 or 16 character in length. If its only 8 characters it represents the lower byte value",
           "detail-desc": [  "1 - Manufacturer reset state",
                             "2 - Device pairing and owner transfer state",
-                            "4 - Provisiong of bootstrap services",
-                            "8 - Provisioning of credential management services",
-                            "16 - Provisioning of access management services",
+                            "4 - Provisioning of bootstrap services",
+                            "8 - Provisioning of security services",
+                            "16 - Provisioning of credential management services",
                             "32 - Provisioning of local ACLs",
-                            "64 - Unused",
-                            "128 - Unused" ]
```

Comment:
Accd to the CR, these are called:

"Initiate Software Version Validation" and "Initiate Secure Software Update".

I had actually defined this change and had it sitting on branch "ocf/1.0/cr/CR-24.SoftwareUpdate". I just forgot to create the PR. doh!